gm4_custom_crafters/data/custom_crafters/functions/main.mcfunction

```diff
@@ -1,5 +1,5 @@
 #crafter creation
-execute as @e[type=item,nbt={Item:{id:"minecraft:crafting_table",Count:1b},OnGround:1b}] at @s if block ~ ~-1 ~ dropper{Items:[{Slot:0b,id:"minecraft:cobblestone",Count:1b},{Slot:1b,id:"minecraft:cobblestone",Count:1b},{Slot:2b,id:"minecraft:cobblestone",Count:1b},{Slot:3b,id:"minecraft:cobblestone",Count:1b},{Slot:5b,id:"minecraft:cobblestone",Count:1b},{Slot:6b,id:"minecraft:cobblestone",Count:1b},{Slot:7b,id:"minecraft:redstone",Count:1b},{Slot:8b,id:"minecraft:cobblestone",Count:1b}]} align xyz unless entity @e[type=armor_stand,tag=gm4_custom_crafter,limit=1,dx=1,dy=1,dz=1] positioned ~.5 ~-1 ~.5 run function custom_crafters:create

```

Comment:
What's so bad about checking the precise block space? I feel like checking the block space is the cleaner way of doing this.

Reply:
1.13 unlocked dx/dy/dz from the block grid, and they detect entity hitboxes rather than execution point. I'm fairly certain the region specified by those deltas overlapped into an adjacent block.